configuration/analyzer_config.json

```diff
@@ -3687,6 +3687,40 @@
       }
     }
   },
+  "ATM Malware": {
```

Comment:
rename to `Yara_Scan_ATM_MALWARE` and place it above the `Yara_Scan_Community` one.

Reply:
Done. I've made these changes sir. Please review :)